Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicRequestBus.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+
+namespace AzToolsFramework
+{
+    namespace Prefab
+    {
+        class PrefabPublicRequests
```

Comment:
Adding a comment like this would be helpful for developers to know when to use the PrefabPublicInterface vs PrefabPublicRequests.
"The primary purpose of this bus is to facilitate writing automated tests for prefabs. It calls PrefabPublicInterface internally to talk to the prefab system. If you would like to integrate prefabs into your system, please call PrefabPublicInterface directly for better performance"

Reply:
Added.